DiscordEnvironmentInfo/index.js

```diff
@@ -0,0 +1,39 @@
+const Plugin = module.parent.require('../Structures/Plugin');
+class DiscordEnvironmentInfo extends Plugin {
+    load() {
+        this.registerCommand({
+            name: 'envinfo',
+            info: 'Shows information about the current environment.',
+            func: this.envinfo.bind(this, true)
+        });
+    }
+    envinfo(args) {
+        const v = new Date().getTime();
+        webpackJsonp([], {[v]: (a, b, d) => {
+            for (let f = 0, g = {};;) {
+                g = d(f++) || {};
+                let h = g._globalOptions;
+                if (h) {
+                    if (h.environment === "ptb") {
+                        var relchannel = "PTB";
+                    }
+                    else {
+                        var relchannel = (h.environment.charAt(0).toUpperCase() + h.environment.slice(1));
+                    };
+                    this.sendLocalMessage(
```

Comment:
I'm reluctant to add due to it being the same value as `process.versions["atom-shell"]`.